webapp/app/forms/steps/lotse_multistep_flow_steps/confirmation_steps.py

```diff
@@ -31,23 +36,37 @@ class Form(SteuerlotseBaseForm):
         confirm_data_privacy = ConfirmationField(label=_l('form.lotse.field_confirm_data_privacy'))
         confirm_terms_of_service = ConfirmationField(label=_l('form.lotse.field_confirm_terms_of_service'))
```

Comment:
Why do we still need these labels in Python?

Reply:
You're absolutely right. We don't.